### DISCUSSION TITLE
How to fix "ValueError: Occupancy fractions not consistent with size of unit cell" when working with supercells?

### DISCUSSION THREAD

**Discussion by mm04926412** [materialsproject/pymatgen#2751] [Q&A]
Hello

In order to get a more representative ordered cell using the orderdisorder transformation, I am first transforming the cell into a supercell. Applying the ordering transformation after the supercell transformation seems to cause the above error. How might I fix this?

Thanks

**Comment by shyuep** [comment]
Pls post an example code with any input files as necessary.
